tests/cli/test_cli_print_url.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import subprocess
+import unittest
+
+
+class TestPrintUrlArgument(unittest.TestCase):
+    """Test CLI for print-url argument."""
+
+    def test_print_url_argument(self):
+        try:
+            args = ['mozdownload',
+                '--url=https://raw.github.com/mozilla/mozdownload/master/README.md',
+                '--print-url']
+            output = subprocess.check_output(args, stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            output = e.output
+
+        url = r'https://raw.github.com/mozilla/mozdownload/master/README.md'
```

Comment:
Something which I already mentioned before is that you should define this URL only once. You can pass it into the call to `mozdownload` and also use it for the assertion check later.